lms/djangoapps/commerce/api/v1/serializers.py

```diff
@@ -17,19 +19,49 @@ def get_identity(self, data):
         except AttributeError:
             return None
 
+    def get_verification_deadline(self, obj):
+        """ Return the course mode's verification deadline.
+
+        Returns None, if the mode is not a verified mode.
+        """
+        if obj.mode_slug in CourseMode.VERIFIED_MODES:
```

Comment:
functionally equal, but i would use the `CourseMode.is_verified_mode` method instead.